templates/torrc-umbrel-sample

```diff
@@ -32,4 +32,8 @@ HiddenServicePort <lnd-rest-port> <lnd-ip>:<lnd-rest-port>
 HiddenServiceDir /var/lib/tor/lnd-grpc
 HiddenServicePort <lnd-grpc-port> <lnd-ip>:<lnd-grpc-port>
 
+# ssh Hidden Service
+HiddenServiceDir /var/lib/tor/ssh
+HiddenServicePort 22 <ssh-ip>:<ssh-port>
```

Comment:
```
HiddenServicePort <ssh-port> <ssh-ip>:<ssh-port>
```

Reply:
> ```
> HiddenServicePort <ssh-port> <ssh-ip>:<ssh-port>
> ```

it was just that the internal port is not configurable else we'll have to manage /etc/ssh/sshd_config too. But as you want, I have updated the PR.